libs/fetcher.js

```diff
@@ -28,6 +28,25 @@ function parseParamValues (params) {
     }, {});
 }
 
+function getErrorResponse(err) {
```

Comment:
Can you add a docblock to this function that describes each property excepted for this function?